private-click-measurement.bs

```diff
@@ -25,120 +25,179 @@ Complain About: missing-example-ids yes
     ISSUE: We may have to add a nonce to the HTTP POST request to prohibit double counting in cases where the user agent decides to retry the request.
 
-If there are multiple ad click attribution requests for the same [=ad click source-destination pair=], the one with the highest [=ad attribution priority=] will be the one sent and the rest discarded.
+If there are multiple ad click attribution requests for the same [=click source-attribute-on pair=], the one with the highest [=optional trigger priority=] will be the one sent and the rest discarded.
 
 Issue: This needs to be reworked to monkeypatch HTML's "follows a hyperlink" algorithm.
 
 </div>
 
-# Ad Click Attribution URLs # {#ad-click-attribution}
+# Well-Known URLs # {#well-known-urls}
+
+## Triggering Event URL ## {#triggering-event-url}
+
+<div algorithm>
+
+Clients <dfn>generate a tiggering event URL</dfn> for |source site| with |trigger data| and |optional trigger priority| by following these steps:
+
+1. Let |url| be the result of [=concatenating=] the strings « `".well-known"`, `"private-click-measurement"`, `"trigger-attribution"`, |trigger data|, |optional trigger priority| » using the separator U+002F (/).
+1. Return the result of calling {{URL(url, base)}} with url |url| and base |source site|.
+
+</div>
+
+## Attribution Report URL ## {#attribution-report-url}
 
 <div algorithm>
 
-Clients <dfn>generate an ad click attribution URL</dfn> for |source| with |attribution data| and |additional data| by following these steps:
+User agents <dfn>generate an attribution report URL</dfn> for |source site| by following these steps:
 
-1. Let |url| be the result of [=concatenating=] the strings « `".well-known"`, `"ad-click-attribution"` |attribution data|, |additional data| » using the separator U+002F (/).
-1. Return the result of calling {{URL(url, base)}} with url |url| and base |source|.
+1. Let |url| be `".well-known/private-click-measurement/report-attribution/"`.
+1. Return the result of calling {{URL(url, base)}} with url |url| and base |source site|.
 
 </div>
 
-# Click source/destination pairs # {#click-source-destination-pairs}
+# Click source/attribute-on pairs # {#click-source-attribute-on-pairs}
 
-An <dfn>ad click source-destination pair</dfn> is a [=tuple=] of two [=sites=]: (source, destination).
+An <dfn>click source-attribute-on pair</dfn> is a [=tuple=] of two [=sites=]: (source, attribute-on).
 
-# Six-bit decimal values # {#six-bit-decimal-values}
+# N-bit decimal values # {#n-bit-decimal-values}
 
+A <dfn>four-bit decimal value</dfn> is a [=string=] for which the [=extract a four-bit decimal value=] algorithm does not return failure.
 A <dfn>six-bit decimal value</dfn> is a [=string=] for which the [=extract a six-bit decimal value=] algorithm does not return failure.
```

Comment:
Is the six-bit algorithm used any more? Can it be removed?

Reply:
Yes, for the optional priority value.